index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Simple Ineterest Calculator</title>
+    <link rel="stylesheet" href="index.css">
+    <script src="index.js"></script>
+</head>
+<body>
+    <div class="maindiv">
+        <h1>Simple Interest Calculator</h1>
+        Principal:
+        <input type="number" id="principal" /><br />
+        Rate: <input type="range" id="rate" min="1" max="20" value="10.25" step="0.25"/><span id="rate_value">10.25</span><br />
```

Comment:
The value 10.25 should change when the range changes. Use JavaScript to control this

Reply:
Thank you I can see the review submission